system/jlib/jfcmp.hpp

```diff
@@ -43,9 +43,9 @@ class jlib_decl CFcmpCompressor : public CSimpleInterfaceOf<ICompressor>
     virtual void setinmax() = 0;
     virtual void flushcommitted() = 0;
 
-    void initCommon()
+    void initCommon(size32_t initialSize)
     {
-        blksz = inma.capacity();
+        blksz = initialSize;
```

Comment:
Does inma.truncate(max) need to be called to actually resize the buffer to a smaller size ?